tests/Phinx/Migration/AbstractMigrationTest.php

```diff
@@ -280,7 +280,7 @@ public function testPostFlightCheckFail()
         $table->addColumn('column1', 'integer', ['null' => true]);
 
         $this->expectException(RuntimeException::class);
-        $this->expectExceptionMessage('Migration has pending actions after execution!');
+        $this->expectExceptionMessage(sprintf('Migration %s_%s has pending actions after execution!', $migrationStub->getVersion(), $migrationStub->getName()));
```

Comment:
Could you make it the set string, instead of having it be filled in dynamically?